python/cudf/cudf/tests/test_copying.py

```diff
@@ -52,3 +53,238 @@ def test_null_copy():
     col = Series(np.arange(2049))
     col[:] = None
     assert len(col) == 2049
+
+
+@pytest.mark.parametrize("copy_on_write", [True, False])
+def test_series_setitem_cow(copy_on_write):
```

Comment:
Since basically every assertion in this test is conditional, it may be better to just split it into two tests.

Reply:
Done, split the tests.